Atomic/syscontainers.py

```diff
@@ -306,6 +346,19 @@ def get_system_containers(self):
             ret.append(container)
         return ret
 
+    def delete_image(self, image):
+        repo = self._get_ostree_repo()
```

Comment:
ok thanks, I have pushed a new version which implements this (and removed the `allow_no_image` parameter).  We don't report the IMAGE DOES NOT EXIST anymore and attempt to delete the image from OSTree after checking there is one.